docs/extensions/readme.md

```diff
@@ -856,8 +856,9 @@ The REST API guidelines define a common pattern for paging through lists of data
 Field Name | Type | Description
 ---|:---:|---
 itemName | `string` | Optional (default: `value`). Specifies the name of the property that provides the collection of pageable items.
-nextLinkName| `string` | Required. Specifies the name of the property that provides the next link (common: `nextLink`). If the model does not have a next link property then specify `null`. This is useful for services that return an object that has an array referenced by `itemName`. The object is then flattened in a way that the array is *directly* returned, no paging is used. This provides a better client side API to the end user.
+nextLinkName| `string` | Required. Specifies the name of the property that provides the next link or a continuation token (common: `nextLink` for the former, `token` for the latter). If the model does not have a next link property then specify `null`. This is useful for services that return an object that has an array referenced by `itemName`. The object is then flattened in a way that the array is *directly* returned, no paging is used. This provides a better client side API to the end user.
 operationName | `string` | Optional (default: `<operationName>Next`). Specifies the name of the operation for retrieving the next page.
+tokenParamName | `string` | Contains the name of the continuation token parameter for the case where `nextLinkName` contains a continuation token instead of a URL. The param needs to be present in the operation used for retrieving the next page, whether that operation has a separate swagger definition or not.
```

Comment:
from @lmazuel: just say "Contains the name of the continuation token parameter that is present in the next operation. Default value is `nextLink`